### PR TITLE
Make Calculator Component

### PR DESCRIPTION
**Pull request by ahangarha**
## Description

I have added:
- Calculator component
- Basic design for Calculator component

### General requirements

*Check items below by adding x between brackets.*

- [x] There are **no linter errors**.
- [x] Followed the **Gitflow**.
- [x] The documentation is **professional**.

### HTML/CSS requirements

- [x] Followed best practices for JavaScript

---
Thank you for taking the time to review this pull request.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,9 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
   return (
```
Comment (julie-ify) [inline src/App.js]: - [x] The `App.js` component should be class-based. Kindly make use of a class component as it is part of the project requirement.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @ahangarha,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

- [x] **[REQUIRED]** Kindly remove all the boilerplate that got created by the CRA (texts, images, and styles). Check the `public` folder of your application and remove all the boilerplate leaving the `index.hml` file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (ahangarha) [reply src/App.js]: I didn't know I had to change App component also. Thanks for pointing it out.

**Review by Meltrust — APPROVED** [review]
Hi @ahangarha,


## Wow, you did it 🎉
Brilliant
## Thank you for making your App component a class one, as requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**